tools/apidoc/build-apidoc.sh

```diff
@@ -62,7 +62,18 @@ set -e
  sed -e 's,%API_HEADER%,Root Admin API,g' "$thisdir/generatetoc_header.xsl" >generatetocforadmin.xsl
  sed -e 's,%API_HEADER%,Domain Admin API,g' "$thisdir/generatetoc_header.xsl" >generatetocfordomainadmin.xsl
 
- python "$thisdir/gen_toc.py" $(find . -type f)
+ WINOS=`uname -s|grep -i WIN`
+ gen_toc_file=''
+ if [ "x$WINOS" != "x" ]; then
+ gen_toc_file="`cygpath -w $thisdir`\\gen_toc.py"
+ else
+ gen_toc_file="$thisdir/gen_toc.py"
+ fi
+ 
+ argfiles=`find . -type f`
```

Comment:
Instead of looping on the found files (argfiles), you can simply give pass it to gen_toc_file?